app/assets/javascripts/rails_admin/ra.nested-form-hooks.coffee

```diff
@@ -50,3 +50,8 @@ $(document).on 'nested:fieldRemoved', 'form', (content) ->
     add_button = toggler.next()
     add_button.addClass('add_nested_fields').html(add_button.data('add-label'))
 
+  # Removing all reuired attributes from deleted child form to bypass browser validations.
+  field.find('[required]').each ->
+    #console.log $(this).attr('name')
+    $(this).removeAttr('required')
+    return
```

Comment:
I just used javascript to coffeescript online converter. I wanted to convert to coffee this:

      field.find('required').each(function() {
            $(this).removeAttr('required')
      });

It returned me that kind of code. It is redundant?